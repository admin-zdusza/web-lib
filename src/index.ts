export { Brand } from './components/brand/Brand';
export { TimeoutError } from './components/timeout-error/TimeoutError'
export { timeoutOccured } from './components/timeout-error/actions-creators';
export { timeoutReducer } from './components/timeout-error/reducers';
