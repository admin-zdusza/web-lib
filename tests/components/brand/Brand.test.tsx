import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { Brand } from "../../../src";

test("should render brand", () => {
  const component = <Brand appName={"Monitoring"} />;
  const { asFragment } = render(component);
  expect(asFragment()).toMatchSnapshot();
});
