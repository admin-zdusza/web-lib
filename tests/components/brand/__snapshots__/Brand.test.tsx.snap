// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render brand 1`] = `
<DocumentFragment>
  <h2>
    Monitoring
    <small
      class="d-none d-md-inline"
      style="color: rgb(119, 119, 119);"
    >
       - z duszÄ…!
    </small>
  </h2>
</DocumentFragment>
`;
